_layouts/synapse-default.html

```diff
@@ -0,0 +1,135 @@
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}</title>
+  {% if page.excerpt %}<meta name="description" content="{{ page.excerpt | strip_html }}">{% endif %}
+  {% if page.tags %}<meta name="keywords" content="{{ page.tags | join: ', ' }}">{% endif %}
+  {% if page.author %}
+    {% assign author = site.data.authors[page.author] %}{% else %}{% assign author = site.owner %}
+  {% endif %}
+  {% include open-graph.html %}
+  {% if site.setCanonicalLink %}
+    <link rel="canonical" href="{{ page.url | absolute_url | replace:'index.html','' | no_trailing_slash }}">
+  {% endif %}
+
+  {% if site.isLocalDev %}
+  <link rel="shortcut icon" type="image/png" href="{{ site.baseurl }}/images/favicon.png">
+  {% else %}
+  <link rel="shortcut icon" type="image/png" href="/v1/config/favicon">
+  {% endif %}
+
+  <link rel="stylesheet" href="{{ site.cdnurl }}{{ site.baseurl }}/css/pygments-github.css">
+  <link rel="stylesheet" href="{{ site.baseurl }}/synapse/dist/synapse.css">
+
+  <link href="{{ site.url }}{{ site.baseurl }}/atom.xml" type="application/atom+xml" rel="alternate" title="{{ site.title }} Atom Feed">
+  <link href="{{ site.url }}{{ site.baseurl }}/sitemap.xml" type="application/xml" rel="sitemap" title="Sitemap">
+
+  <meta name="HandheldFriendly" content="True">
+  <meta name="MobileOptimized" content="320">
+  <meta name="viewport" content="width=device-width, initial-scale=0.75">
+  <meta http-equiv="cleartype" content="on">
+
+  {% if site.webserver %}
+    {% assign webserver = site.webserver %}
+  {% else %}
+    {% assign webserver = "https://algorithmia.com" %}
+  {% endif %}
+</head>
+
+<body>
+  <div class="syn-page syn-page-drawer-navigation">
+    <header class="syn-page-header syn-header-brand">
+      <div class="syn-page-header-content">
+        <button onclick="['nav-open', 'nav-collapsed'].forEach((a) => document.querySelector('.syn-page-header').classList.toggle(a))" class="syn-btn text icon rounded grow theme-light no-shadow" data-v-3352aef0="" data-v-19ae9fc3="">
```

Comment:
:bulb: Going to create a ticket to ensure we bring over the "persistent close/open state" for the menu.